redisinsight/ui/src/components/csrf/Csrf.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+
+import apiService, { setCSRFHeader } from 'uiSrc/services/apiService'
+import PagePlaceholder from '../page-placeholder'
+
+const csrfEndpoint = () => process.env.RI_CSRF_ENDPOINT || ''
+
+interface CSRFTokenResponse {
+  token: string;
+}
+
+const Csrf = ({ children }: { children?: ReactElement }) => {
+  const [loading, setLoading] = useState(true)
```

Comment:
I think default value should be `false`, and just call `setLoading(true)` in the `fetchCsrfToken()`

Reply:
done